assets/questions/rle.js

```diff
@@ -26,26 +26,33 @@ globalData.questions['rle'] = {
   },
   auxiliaryCode: {
     python:
-`def skipEncodingAtEndOfString(word):
-    pass
+`class AuxiliaryCode(object):
+    @classmethod
+    def skipEncodingAtEndOfString(cls, word):
+        pass
 
-def ignoreStringLengthWhenEncoding(word):
-    pass
+    @classmethod
+    def ignoreStringLengthWhenEncoding(cls, word):
+        pass
 
-def failToDemarcateBeginningOfEncodedChunk(word):
-    pass
 
-def DecodeEncodedString(word):
-    pass
+    @classmethod
+    def failToDemarcateBeginningOfEncodedChunk(cls, word):
+        pass
+
+
+    @classmethod
+    def DecodeEncodedString(cls, word):
+        pass
 `
   },
   prompts: [{
     instructions: [
       [
-        'Implement the encode function. It takes a string as input and ',
-        'returns an encoding of the string where long runs of characters are ',
-        'replaced by <# characters>x<character>. For example, "abcccccd" ',
-        'should be encoded as "ab5xc".'
+        'In this question, you\'ll implement the encode function. It takes a ',
+        'string as input and returns an encoding of the string where long ',
+        'runs of characters are replaced by <# characters>x<character>. For ',
```

Comment:
That's part of the problem that the learner has to figure out. The encoding scheme is intentionally bad, so the student needs to figure out ways to prevent that. I don't want to put a solution to this problem in this comment chain, so I'm happy to discuss this offline, but the burden of making the encodings distinct is on the learner.

Though technically, for your example, the first string should remain 7xx and the second string should be 72xx. :)